StackExchange.Profiling.Wcf/WcfRequestProfilerProvider.cs

```diff
@@ -203,5 +204,21 @@ private void SetCurrentProfiler(MiniProfiler profiler)
 
             context.Items[WcfCacheKey] = profiler;
         }
+
+        public override Timing GetHead()
```

Comment:
this is more to support the new methods, the implementation is not very advanced here since i don't really care about wcf